platform/email/src/main/java/org/motechproject/email/web/EmailController.java

```diff
@@ -9,10 +9,10 @@
 import org.motechproject.email.constants.EmailRolesConstants;
 import org.motechproject.email.domain.DeliveryStatus;
 import org.motechproject.email.domain.EmailRecord;
-import org.motechproject.email.domain.EmailRecords;
 import org.motechproject.email.service.EmailAuditService;
 import org.motechproject.mds.query.QueryParams;
 import org.motechproject.mds.util.Order;
+import org.motechproject.mds.web.domain.Records;
```

Comment:
If you want to use some common class across modules, you should place it in one of the commons module